src/components/Attachment.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+	<li class="list-item--attachment" :class="{'error' : attachment.error }">
+		<div class="attachment-preview">
+			<img v-if="attachment.imageBlobURL !== false" :src="attachment.imageBlobURL" class="attachment-preview-image">
+			<img v-else-if="attachment.hasPreview" :src="previewURL" class="attachment-preview-image">
+			<img v-else :src="getIcon" class="attachment-preview-image">
+			<span v-if="attachment.type === 'cloud'" class="cloud-attachment-icon">
+				<Cloud :size="16" />
+			</span>
+		</div>
+		<div class="attachment-inner">
+			<span class="new-message-attachment-name">
+				{{ attachment.displayName ? attachment.displayName : attachment.fileName }}
+			</span>
+			<span v-if="!attachment.finished" class="attachments-upload-progress">
+				<span class="attachments-upload-progress--bar" :style="&quot;width:&quot; + attachment.percent + &quot;%&quot;" />
+			</span>
+			<span v-else class="new-message-attachment-size">{{ attachment.sizeString }}</span>
+		</div>
+		<button @click="onDelete(attachment)">
+			<Close :size="24" />
+		</button>
+	</li>
+</template>
+
+<script>
+import { generateUrl } from '@nextcloud/router'
+import Close from 'vue-material-design-icons/Close'
+import Cloud from 'vue-material-design-icons/Cloud'
+
+export default {
+	name: 'Attachment',
+	components: {
+		Close,
+		Cloud,
+	},
+	props: {
+		bus: {
+			type: Object,
+			required: true,
+		},
+		attachment: {
+			type: Object,
+			required: true,
+		},
+		uploading: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	data() {
+		return {
+			progress: 0,
+			sizeString: '',
+			finished: false,
+		}
+	},
+	computed: {
+		previewURL() {
+			if (this.attachment.hasPreview && this.attachment.id > 0) {
+				return generateUrl(`/core/preview?fileId=${this.attachment.id}&x=100&y=100&a=0`)
+			}
+			return ''
+		},
+		getIcon() {
+			return OC.MimeType.getIconUrl(this.attachment.fileType)
+		},
+		extention() {
+			return this.attachment.fileName.split('.').pop()
+		},
+	},
+	methods: {
+		onDelete(attachment) {
+			this.$emit('on-delete-attachment', attachment)
+		},
+	},
+
+}
+</script>
+
+<style lang="scss" scoped>
+
+.list-item--attachment {
+	width: calc(50% - 20px);
+    box-sizing: border-box;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    margin: 10px;
+    flex-wrap: wrap;
+
+	&.error {
+		color:red;
+		opacity: 0.5;
+		cursor: not-allowed;
```

Comment:
```suggestion
```

This cursor might confuse users. You already added a red attachment title which is  sufficient.